interhost.py

```diff
@@ -262,6 +270,94 @@ def main_align_mafft(args):
     return 0
 __commands__.append(('align_mafft', parser_align_mafft))
 
+# =======================
+# ***  multichr_mafft  ***
+# =======================
+
+def parser_multichr_mafft(parser=argparse.ArgumentParser()):
+    parser.add_argument('inFastas', nargs='+',
+        help='Input FASTA files.')
+    parser.add_argument('--outDirectory', default=os.getcwd(),
+        help='Location for the output files (default is cwd: %(default)s)')
+    parser.add_argument('--outFilePrefix', default="singlechr",
+        help='Prefix for the output file name (default: %(default)s)')
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--localpair', default=None, action='store_true',
```

Comment:
Sure, I'll combine them. I had them separate because I wasn't sure if the defaults might be different for the multichr version.